samples/llvm/brainfuck.cr

```diff
@@ -56,7 +56,7 @@ class Read < Instruction
     current_cell_ptr = builder.gep program.cell_type, program.cells_ptr, cell_index, "current_cell_ptr"
 
     getchar = program.mod.functions["getchar"]
-    input_char = builder.call getchar, "input_char"
+    input_char = builder.call program.func_types["getchar"], getchar, "input_char"
```

Comment:
Not a biggie, but the `builder.call ...` part might be abstracted in its own function.